pkgs/development/python-modules/pylatex/default.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, ordered-set
+}:
+
+buildPythonPackage rec {
+  pname = "pylatex";
+  version = "1.3.4";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```
Could you please use `pyproject = true` instead?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters

While waiting for the merge, the recommended writing style has changed. Sorry for the extra work.

---

Done!